doc/userguide/performance/runmodes.rst

```diff
@@ -12,7 +12,7 @@ a queue. Packets will be processed by one thread at a time, but there
 can be multiple packets being processed at a time by the engine (see
 :ref:`suricata-yaml-max-pending-packets`). A thread can have one or
 more thread-modules. If they have more modules, they can only be
-active one a a time.  The way threads, modules and queues are arranged
+active one at a time.  The way threads, modules and queues are arranged
```

Comment:
There is another one :
```
git grep " a a "
doc/userguide/performance/hyperscan.rst:**Note:** Hyperscan can take a a long time to build/compile.
doc/userguide/performance/runmodes.rst:active one a a time.  The way threads, modules and queues are arranged
```

Reply:
Oh, thanks for pointing that out! I'll fix that too.
Should I squash the commits into a single one, or is it okay to have two commits in this PR?
Do I need to create a new PR (as suggested earlier), or can I just update this one? 
Also, do I need to create a new ticket for this additional change, or is it fine to update the current one?
just to confirm, I’ve signed the CLA.